sopel/trigger.py

```diff
@@ -331,6 +343,15 @@ class Trigger(str):
         The :attr:`COMMANDS_WITH_CONTEXT` attribute lists IRC commands for
         which ``sender`` can be relied upon.
 
+    """
+    status_prefix = property(lambda self: self._pretrigger.status_prefix)
+    """The prefix used for the :attr:`sender` for status-specific messages.
+
+    :type: str | None
+
+    This will be ``None`` by default. If a message is sent to a channel, it may
+    have a status prefix for status-specific messages. In that case, the prefix
+    is removed from the channel's identifier and set to this attribute.
```

Comment:
```suggestion
    is removed from the channel's identifier and saved to this attribute.
```

(Also, cross-ref `identifier` to the class?)

Reply:
I tried to do `Channel.name` and it was ugly and not really helpful. So I went for `:attr:sender` which is more relevant to the context.